Gruntfile.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+module.exports = function (grunt) {
+    grunt.config.init({
+        lintAll: {
+            sources: {
+                js:    ["./src/js/**/*.js", "tests/js/**/*.js", "./*.js", "!./src/lib/**/*.js"]
```

Comment:
You need to add configuration blocks for your markdown content, and any JSON files you end up with.  This is why your overly long lines of markdown didn't throw a linting error when running `grunt lint`.  I can see that the docs for `gpii-grunt-lint-all` are not great, as their examples only include a `js` entry.  Check out the [Gruntfile for that project|https://github.com/GPII/gpii-grunt-lint-all/blob/master/Gruntfile.js#L18] for a fuller example of what you need to include.